Update getting-started.mdx added a comment for the linux users. (#249)
kubeshop/botkube-docs: docs/cli/getting-started.mdx

<TabItem value="linux" label="Linux">

Download the Botkube CLI binary and move it to a directory under your `$PATH`:

> You might need to use "sudo" to run the cmd mv as it is moving the binary file under the directory /usr/local/bin/ 

Reviewer: ```suggestion
You may need to use `sudo` to run the `mv` command as it moves the binary file to the directory `/usr/local/bin/`.
```
